checkstyle.xml

```diff
@@ -67,7 +67,10 @@
         <module name="LineLength">
             <property name="max" value="120"/>
         </module>
-        <module name="MethodLength"/>
+	<module name="MethodLength">
```

Comment:
TotemProtocolDecoder.decode() is 155 lines long.  I didn't want to materially change the file by breaking decode() up into smaller pieces.  If you insist, I will do it.